### DISCUSSION TITLE
请问nullcontext没定义是咋回事呀

### DISCUSSION THREAD

**Discussion by ZJ-CAI** [PaddlePaddle/PaddleSpeech#1044] [Q&A] [answered]
File "/home/ubuntu/czj/PaddleSpeech/paddlespeech/s2t/exps/deepspeech2/model.py", line 93, in train_batch
    context = nullcontext
NameError: name 'nullcontext' is not defined

**Comment by ZJ-CAI** [comment, marked as answer]
原来是python3.6版本太低

**Reply by Jackwaterveg** [reply]
请使用3.7+的python哦
